PharoLauncherTutorial/pharolauncher_tutorial.pillar

```diff
@@ -0,0 +1,126 @@
+
+
+
+!! Downloading/Installing PharoLauncher
+
+As discussed earlier about the rapid evolvement of Pharo , kindly check out *http://www.pharo.org/download* to get the latest download/install instructions for Pharo Launcher.
+
+!!!Linux Ubuntu:
+(*http://pharo.org/download#ubuntu*)
+
+Ubuntu users can use the dedicated ppa to install Pharo
+
+[[[
+sudo add-apt-repository ppa:pharo/stable
+sudo dpkg --add-architecture i386
+sudo apt-get update
+sudo apt-get install pharo-launcher
+]]]
+
+If you don't have the add-apt-repository command, install the software-properties-common package and try again.
+
+If you are on a server (no GUI), you can get a Pharo Virtual Machine by installing pharo-vm-core.
+
+On Ubuntu, the Launcher is installed as ''/usr/bin/pharo'', so you can type pharo on the command line to start it.
+
+!!!ArchLinux :
+(*http://lists.pharo.org/pipermail/pharo-users_lists.pharo.org/2014-March/010932.html*)
+[[[
+$ yaourt pharo-vm-latest
+$ pharo /path/to/your/image
+]]]
+
+There's also a pharo-launcher package that depends on pharo-vm:
+[[[
+$ yaourt pharo-launcher
+$ pharo-launcher
+]]]
+
+!!!Windows:
+Download and install the executable from the link provided here (*http://files.pharo.org/platform/launcher/*)
+
+
+!!!MacOS:
+Use the link (*http://files.pharo.org/platform/launcher/*)to install PharoLauncher on Mac system.
+
+After installation , you'll observe that the Launcher is installed in /Applications. 
+
+
+!!Using PharoLauncher
+
+Launch the Pharo launcher image using the platform-specific VM.
+
+The image below depicts how a PharoLauncher looks like when it is opened.
+
+
+
++Pharo Launcher>file://figures/pharolauncher.png|width=100|label=obj+
+
+
+
+The first screen has been divided into two parts.
+The first part 'Existing Images' displays the images already created by the user. Initially after the installation the left side with local images is empty.
+Whereas the the other 'Templates' displays the image templates from various resources available for download from the internet.
+The 'Existing Images' section has 3 buttons : Launch, Delete and Refresh.
+The 'Templates' section has 2 buttons : Create Image and Refresh.
+At the bottom of the launcher we have the buttons for quit and settings.
+
+Select the image you wish to work on from the list and the sources provided in the 'Templates' section and download it. 
+For instance you can download "Pharo4.0 (beta)” from the options provided which is the latest image as of today. By clicking on the 'Create Image' button at the top right corner.
+Note that also the images from contribution CI are available.
+So you can easily download "Artefact", "Moose", ... images according to your choice.
+
+It will download the image into a specific directory somewhere in your users home directory.
+Each image gets an own folder. Use the "Show in folder" menu item  to open this location.
```

Comment:
I would be specific that it's the /context/ menu and maybe show a cropped screen shot of the menu and existing images pane